### PR TITLE
Workflows doc

### PR DESCRIPTION
**Pull request by mattmanning**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by mattmanning** [comment]
[Document Console Workflows](https://trello.com/c/s2iqADaG/774-document-console-workflows)

File: _console/workflows.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+[screenshot]
+
+## Suggested Workflows
+
+As mentioned above, workflows can be used along with other tools that set commit statuses to build Continuous Integration pipelines. An example of one of these pipelines might look like:
+
+1. Configure TravisCI to run on pull requests and set the commit status when finished
+2. Create workflow that triggers on your repo's master branch
+3. Define a task to build an promote your staging app
```
Comment (ianmalott) [inline _console/workflows.md]: an -> and

File: _console/workflows.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+4. Define a second task to build (but not promote) your production app
+
+With this setup, a merge into your master branch will automatically execute everything needed right up to going live in production. If you are satisfied with the state of your staging app when the workflow completes, you can simply click "Promote" on your production app in Console to complete the process.
+
+For a fully automated production pipeline, enable promotion in step 4. Since a failure of any task in a workflow halts execution for remaning tasks, you can be sure that your production app will only be promoted if the staging app was successfully promoted (i.e. all containers were rolled out and passed health checks).
```
Comment (ianmalott) [inline _console/workflows.md]: remaning -> remaining

File: _console/workflows.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
+title: "Workflows"
+---
+
+[Convox Console](https://console.convox.com) supports the creation of "workflows" – series of tasks that are executed when triggered by user-defined actions. Workflows make it easy to automate continuous delivery of your applications.
```
Comment (ianmalott) [inline _console/workflows.md]: Looking for a way to highlight that Workflows is ultimately about more than automating CD. For example:

```diff
- Workflows make it easy to automate continuous delivery of your applications.
+ The first version of Workflows makes it easy to automate continuous delivery of your applications. Eventually, Workflows will enable you to automate ________.
```
Brainstorming ideas for the blank:

- all kinds of things
- even more of your development processes
- everything but writing application code

File: _console/workflows.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+Next, choose a trigger for the workflow. This is the action that will kick off execution of the workflow tasks. Begin by choosing an integration. GitHub and GitLab integrations are currently available, and can be managed on the Integrations tab.
+
+Once you've chosen an integration, define the rest of the trigger. For GitHub and GitLab, this means choosing a repository and a branch.
+
+![](/assets/images/docs/workflows/trigger.png)
+
+When commits are pushed to the specified repository branch, workflow tasks are triggered.
+
+<div class="block-callout block-show-callout type-info" markdown="1">
+  If the status of the latest commit is *pending* or *failing*, the workflow will not be triggered. This allows you to build integration testing gates, for example, into your workflows using tools such as CircleCI or TravisCI which can set commit statuses.
```
Comment (awsmsrc) [inline _console/workflows.md]: this is only true for github and only if they have required status

File: _console/workflows.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+![](/assets/images/docs/workflows/manage.png)
+
+## Workflow History
+
+The execution of a workflow is referred to as a "job". Click the Jobs tab to view a list of previously executed workflows. A red or green indicator will tell you whether the job completed successfully.
+
+![](/assets/images/docs/workflows/jobs.png)
+
+Click the timestamp link associated with a job to see more detailed information including logs.
```
Comment (awsmsrc) [inline _console/workflows.md]: some jobs don't have workflows, these are usually manually triggered jobs such as a rack install

Comment (mattmanning) [inline _console/workflows.md]: Good to know. I will make a second pass after this PR to elaborate on jobs.

File: _console/workflows.md (hunk 3 above)
Comment (mattmanning) [inline _console/workflows.md]: Removed specifics for now. We can flesh out this section when we have more options for types of workflows.

File: _console/workflows.md (hunk 2 above)
Comment (mattmanning) [inline _console/workflows.md]: fixed

File: _console/workflows.md (hunk 1 above)
Comment (mattmanning) [inline _console/workflows.md]: fixed

File: _console/workflows.md (hunk 4 above)
Comment (mattmanning) [inline _console/workflows.md]: removed for now

**Pull request merged**